docs/book/features/block_volume.md

```diff
@@ -376,6 +378,46 @@ cluster. Make sure to mention `pv.kubernetes.io/provisioned-by: csi.vsphere.vmwa
         Events:                <none>
     ```
 
+### Statically Provision a Block Volume in vSphere with Kubernetes<a id="static_volume_provisioning_vsphere_with_k8s"></a>
+
+In vSphere 7.0, Project Pacific does not support static volume provisioning in vSphere with Kubernetes aka supervisor cluster. Static volume provisioning is a process where an existing volume on the storage is made available in the Kubernetes cluster. There are use cases where we want to import/bring in existing volumes into supervisor cluster. For example, customer wants to migrate OpenShift managed vanilla cluster to Tanzu Kubernetes Grid Service for the benefits offered by vSphere with Kubernetes.
+
+A new CnsRegisterVolume API is introduced in vSphere with Kubernetes where a vSphere Admin would be able to import this volume into supervisor namespace. CnsRegisterVolume API objects are namespaced objects and once created successfully it would create a PVC in that namespace and a PV bound to that PVC.
```

Comment:
We do not have UI for vSphere admin to import the volume.

Reply:
I have rephrased it slightly.